packages/palette-docs/content/docs/elements/layout/GridColumn.mdx

```diff
@@ -0,0 +1,72 @@
+---
+name: GridColumn
+---
+
+(WIP text) The Grid element is a layout container for rendering a Grid
```

Comment:
@dzucconi - would you mind pushing a commit that updates this text?

Reply:
👍